README.md

```diff
@@ -1,2 +1,66 @@
-# my-portfolio
-Building My Portfolio Website
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Porfolio Website
+
+> Building My Portfolio Website
+
+![screenshot](./app_screenshot.png)
+
+This is my portfolio website to showcase my skills and projects I have done as a software engineer. It is motivate by my Microverse program start on the 5th, July, 2021.
+
+## Built With
+
+- HTML
+- CSS
+
+## Live Demo
+
+[Live Demo Link](https://livedemo.com)
+
```

Comment:
- [ ] Include the Live link demo if you have it.

Reply:
Thank you so much, Emmyn5600 :100: 

The problem was that my CSS file had its file path as '/css/styles.css' instead of './css/styles.css'. That actually made the whole page load without loading the CSS file online. I was actually working on my machine :smile: 

I have also added a live link in the readme for your information and necessary actions.

All other recommended actions from you have been taken. :pray: 

I will go ahead and resubmit a review request :running_man: